components/Order/interfaces.rst

```diff
@@ -0,0 +1,124 @@
+Interfaces
+==========
+
+Model Interfaces
+----------------
+
+.. _component_order_model_adjustment-interface:
+
+AdjustmentInterface
+~~~~~~~~~~~~~~~~~~~
+
+This interface provides you with basic management of an adjustment's properties.
```

Comment:
For every model interface, I'd say something like this:

"This interface should be implemented by model representing a single Adjustment." or "This interface is implemented by Product model." or "This interface must be implemented by objects representing a Taxonomy.". Something like this.

What you proposed here is quite obvious for every interface.